source/usage-examples/find.txt

```diff
@@ -0,0 +1,164 @@
+=======================
+Find Multiple Documents
+=======================
+
+.. default-domain:: mongodb
+
+Overview
+--------
+
+``collection.find()`` lets you
+`query <https://docs.mongodb.com/manual/tutorial/query-documents/#read-operations-query-argument>`_
+for multiple documents in MongoDB. You can pass ``find``
```

Comment:
nit: find is a function, so I'd notate it as `find()` rather than `find`

Reply:
I don't have a strong opinion on this, but ``find`` occurs a lot on this page, as you might expect. I think it feels a little disruptive to see the extra parens, but that's definitely a personal feeling. Thoughts, @nlarew ?